mt_with_external_memory/README.eng.md

```diff
@@ -36,15 +36,15 @@ Fig 1. state vector in LSTM as memory path
 
```

Comment:
line 19: With memory, cognition can be coordinated in time, making complex cognition (deducing, planning, and other mental activities that are different from static perception) possible.